store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -38,10 +39,39 @@ public class OwaspHtmlSanitizer implements Callable<String> {
     private String vHost;
 
     public OwaspHtmlSanitizer(String html, boolean neuterImages, String vHost) {
-        this.html = html;
+    	// temporary fix to check extra tags & remove
+        String formatHtml = checkUnbalancedTags(html);
+        this.html = formatHtml;
         this.neuterImages = neuterImages;
         this.vHost = vHost;
     }
+    
+    // method to check extra tags which may have started , not ended & remove ( removing unbalanced tags )
+    /* in tags array we can add multiple tags , so for each tag write start & end with a & separator like (<!--&-->)
+     defining start & end with split & spliting html with start tag & checking it has end or not
+     if end is not there for particular start then it will skip or if end is there it will add as it is */
+    private String checkUnbalancedTags(String html) {
```

Comment:
indentation issues through out the method

Reply:
changed